docs/user-manual/en/resource-adapter.md

```diff
@@ -0,0 +1,130 @@
+Config options for the inbound ConnectionFactory
+[ActiveMQConnectionFactory](https://github.com/apache/activemq-artemis/blob/main/artemis-jms-client/src/main/java/org/apache/activemq/artemis/jms/client/ActiveMQConnectionFactory.java)
+- `brokerUrl` url to broker.
+- `cacheDestinations` by the jms session
+#### ConnectionManager
+You can't configure any properties.
+### ActivationSpec
+Config options 
+[ActiveMQActivationSpec](https://github.com/apache/activemq-artemis/blob/main/artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java)
+
+In the activation spec you can configure all the things you need to get messages consumed from artemis.
+- `useJndi` true if you want lookup destinations via jndi.
+- `connectionFactoryLookup` the jndiName of the connectionFactory, used by this activation spec. You can reference 
+  an existing ManagedConnectionFactory or specify another.
+- `jndiParams` for the InitialContext. key value pairs, like a=b;c=d;e=f
+- `destination` name or jndi reference of the destination
+- `destinationType` javax/jakarta.jms.Queue or javax/jakarta.jms.Topic
+- `messageSelector` sql where clause syntax to filter messages to your message driven bean
+#### Only for topic message consumption
+- `subscriptionDurability` Durable / NonDurable
+- `subscriptionName` artemis hold all messages for this name if you use durable subscriptions 
+
+## Logging
+With the package `org.apache.activemq.artemis.ra` you catch all ResourceAdapter logging statements.
+
+## Example for WLP / OpenLiberty
+sample for your server.xml, in this case, with the jms 3.0 API. For a runnable sample, take also a look in the 
+examples/features/sub-modules/wlp directory. 
+```xml
+<server>
+  <resourceAdapter id="artemis" location="${wlp.user.dir}shared/lib/artemis.rar">
+      <properties.artemis />
+  </resourceAdapter>
+  
+  <!-- dont't use the jmsConnectionFactory element -->
+  <connectionFactory jndiName="jms/sampleArtemisConnectionFactory">
+      <properties.artemis />
+      <connectionManager /> <!-- not implemented by Artemis-->
+  </connectionFactory>
+  
+  <jmsActivationSpec id="your-app/QueueListener">
+      <properties.artemis useJndi="true" destination="jms/sampleQueue" destinationType="jakarta.jms.Queue"
+          connectionFactoryLookup="jms/sampleArtemisConnectionFactory" />
+  </jmsActivationSpec>
+  
+  <jmsActivationSpec id="your-app/TopicListener">
+      <properties.artemis clientId="aClientId" useJndi="true" destination="jms/sampleTopic"
+          destinationType="jakarta.jms.Topic" subscriptionDurability="Durable" subscriptionName="sampleDurableSubscriberName"
+          connectionFactoryLookup="jms/asyncAppConnectionFactory" />
+  </jmsActivationSpec>
+  
+  <jmsTopic jndiName="jms/sampleTopic">
+      <properties.artemis Address="a_sample_adress" />
+  </jmsTopic>
+  
+  <jmsQueue jndiName="jms/sampleQueue">
+      <properties.artemis Address="another_sample_adress" />
+  </jmsQueue>
+</server>
+```
```

Comment:
I dont see a need to embed this in the docs. It wont apply to many people and is just going to go stale. Seems like it would be better to just link to the example, which has it. Perhaps at the same time as mentioning other examples above.